contributing.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
```

Comment:
ah interesting! Would it be like I add a commit to your branch? what's the experience like - I'd want to make it clear what I'd be changing